cf-uaa-lib.gemspec

```diff
@@ -31,8 +31,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   # dependencies
-  s.add_dependency 'multi_json', '>= 1.12.1', '< 1.16'
```

Comment:
why remove multi_json as well? I was looking at the commit history and saw that it was explicitly added again here: https://github.com/cloudfoundry/cf-uaa-lib/commit/dbc424981eda7b04cd2db3bf49a8db1c67277ea7

Reply:
I suspect this may be some sort of compatibility thing with `json` / `json_pure` though I'm not sure....

per the [multi-json readme]( https://github.com/intridea/multi_json) it looks like in order to use it one should require `multi_json` and use it via `MultiJson...`

Doing a search in cf-uaa-lib I don't see any uses of `MultiJson`:
https://github.com/search?q=repo%3Acloudfoundry%2Fcf-uaa-lib%20MultiJson&type=code

... and the only results for `multi_json` are in the `LICENSE` file.